ansible/playbooks/sap-hana-cluster.yaml

```diff
@@ -40,6 +43,11 @@
     - name: Load SAP HANA variables
       ansible.builtin.include_vars: ./vars/hana_vars.yaml
 
+    - name: Re-set HANA Angi vars
```

Comment:
Does this refer to line 24, right? Var name is the same but value has to be different in case of Angi? Do I correctly understand it?

Reply:
Yes